index.js

```diff
@@ -138,6 +140,36 @@ function findAllDependencies(file, knownDependencies) {
   });
 }
 
+// write compiled Elm to a string output
+// callback takes (err, data), where data is a Buffer of
```

Comment:
`findAllDependencies` [returns a Promise](https://github.com/rtfeldman/node-elm-compiler/blob/master/index.js#L73) instead of taking a callback, and I'd like to keep consistent with that!